shell/platform/android/io/flutter/embedding/engine/systemchannels/ScribeChannel.java

```diff
@@ -62,6 +63,11 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull MethodChannel.Result
 
   private void isStylusHandwritingAvailable(
       @NonNull MethodCall call, @NonNull MethodChannel.Result result) {
+    if (Build.VERSION.SDK_INT < API_LEVELS.API_34) {
+      result.error("error", "Requires API level 34 or higher.", null);
```

Comment:
Does this mean that the dart side of the plugin needs to check if it is running on android and check the api level?

Reply:
I guess in practice this probably means that you want to call this wrapped in a `try/catch` in Dart. My thought process is that this method channel method is just a proxy for InputMethodManager.isStylusHandwritingAvailable. If I'm not even able to call that, I should error, rather than succeed with `false`, which the app developer might interpret to mean that InputMethodManager.isStylusHandwritingAvailable returned false.

If that sounds reasonable then I'll at least make sure this is clearly documented in the framework.

Reply:
Update here: https://github.com/flutter/engine/pull/52943#issuecomment-2423177793